crates/sparrow-runtime/src/execute/output/pulsar.rs

```diff
@@ -66,9 +66,9 @@ pub(super) async fn write(
     };
 
     let topic_url = format_topic_url(&pulsar)?;
-    let output_schema =
-        pulsar_schema::get_output_schema(schema).change_context(Error::SchemaSerialization)?;
-    let formatted_schema = pulsar_schema::format_schema(output_schema.clone())
+    let output_schema = streams::pulsar::schema::get_output_schema(schema)
+        .change_context(Error::SchemaSerialization)?;
+    let formatted_schema = streams::pulsar::schema::format_schema(output_schema.clone())
```

Comment:
What is a "formatted" schema? It seems like this could probably have a more descriptive name.

Reply:
Yeah, this is the `avro_schema` actually. Changed